contents/docs/deployment/deploy-aws-clickhouse.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Deploying to AWS with ClickHouse
+sidebar: Docs
+showTitle: true
+---
+
+## Why ClickHouse on AWS?
+
+
+This deployment mechanism is targeted towards larger installations (above 2M events/month) which would run into performance and scalability issues using our normal postgres-based deployments.
+
+As opposed to the open source version, this deployment uses ClickHouse instead of PostgreSQL as the underlying database. ClickHouse is [a highly scalable database that was designed for performant analytical queries](https://clickhouse.tech/).
+
+> Warning: This deployment mechanism is still experimental and is not yet publicly accessible
```

Comment:
`and you will currently need a license to use clickhouse. To get a license, please message [sales@posthog.com](mailto:sales@posthog.com)`